### PR TITLE
Allow notify-on to be in config

### PR DESCRIPTION
**Pull request by dhpollack**
Small update to the notify on feature to allow it to be set in the config.

### CODE REVIEW

**Review by savingoyal — CHANGES_REQUESTED** [review]
added a comment

File: metaflow/metaflow_config.py [hunk 1]
```diff
@@ -308,6 +308,8 @@
 ARGO_EVENTS_INTERNAL_WEBHOOK_URL = from_conf(
     "ARGO_EVENTS_INTERNAL_WEBHOOK_URL", ARGO_EVENTS_WEBHOOK_URL
 )
+ARGO_EVENTS_NOTIFY_ON_ERROR = from_conf("ARGO_EVENTS_NOTIFY_ON_ERROR", False)
```
[the remark this answers is not in the document]
Comment (dhpollack) [reply metaflow/metaflow_config.py]: Is there documentation on `METAFLOW_ARGO_WORKFLOWS_CREATE_NOTIFY_ON_SUCCESS`?  I can't seem to find it nor do I see it in the code.

Made the change to use the `ARGO_WORKFLOWS` prefix.

[the remark this answers is not in the document]
Comment (savingoyal) [reply metaflow/metaflow_config.py]: It's a feature of click. all cli arguments for metaflow are supported this way already.

**Comment by dhpollack** [comment]
If this is possible without this PR then we can close this.

**Pull request closed without merging**